tutorials/luigi-app-react/luigi-app-react.md

```diff
@@ -191,24 +189,34 @@ In this step, you will create another HTML file which will serve as a template f
 
 In this step, we configure `webpack` and adjust dependencies in order to make it easier to develop and build the app.
```

Comment:
Use You and not the royal We.(unless, in rare cases, you are referring to SAP).

Reply:
Thank you and sorry for missing this; I just pushed the changes.